courses/templates/catalog.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load i18n static render_bundle %}
+
+{% block title %}{% trans "MIT xPro | Course Catalog" %}{% endblock %}
```

Comment:
Do we need to translate anything for this project? I'm not sure it's worth doing here if we aren't doing it elsewhere.